.github/workflows/cli.yml

```diff
@@ -0,0 +1,67 @@
+name: run CLI
+on:
+  push:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  basic-usage:
+    name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+    - name: building
+      run: |
+        npm install ts-node -g
+        npm install
+        npm run build
+    - name: create workspace
+      run: mkdir -p myworkspace
+    - name: test client info
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js -v
+        ../packages/cli/dist/bin/index.js --version
+        ../packages/cli/dist/bin/index.js -h
+        ../packages/cli/dist/bin/index.js --help
+    - name: test client init
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js init --help
+    - name: test client run
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js run --help
+    - name: test client log
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js log --help
+    - name: test client board
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js board --help
+    - name: test client plugin-dev
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js plugin-dev --help
+    - name: test client dataset
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js dataset --help
+    - name: test client bip
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js bip --help
+    - name: test client serve
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js serve --help
```

Comment:
I think we should merge the above --help commands into one, and add more jobs for other commands.

Reply:
Good idea.

Reply:
See my https://github.com/alibaba/pipcook/pull/137#issuecomment-621615230, we need some tests for these cases.

Reply:
BTW We should make sure `pipcook init` and `pipcook run <pipeline>` are working at least.

Reply:
Yeah, I will add that case :)